src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -1026,14 +1044,30 @@ protected bool CheckType(TexlNode node, DType nodeType, DType expectedType, IErr
             return false;
         }
 
+        protected bool TryGetSingleColumn(DType type, TexlNode arg, IErrorContainer errors, out TypedName column)
+        {
+            IEnumerable<TypedName> columns;
+
+            column = new TypedName();

```

Comment:
>new TypedName(); [](http://example.com/codeflow?start=21&length=16)

likely should be null/default, not new TypedName()

Reply:
Done.